src/data/repositories/ConsoleLoggerRepository.ts

```diff
@@ -4,6 +4,10 @@ import { DefaultLog } from "../../domain/entities/Log";
 import { LoggerRepository } from "../../domain/repositories/LoggerRepository";
 
 export class ConsoleLoggerRepository implements LoggerRepository {
+    batchLog(logs: DefaultLog[]): FutureData<void> {
+        logs.map(log => this.log(log));
+        return Future.success(undefined);
+    }
     log(log: DefaultLog): FutureData<void> {
```

Comment:
prettier does not force us, but multi-line methods are typically separated by a blank line, to improve readability (this does not apply to the interface, which are all one-liners)